ccs/dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml

```diff
@@ -0,0 +1,33 @@
+﻿# This Deployment manifest defines:
+# - single-replica deployment of the container image, with label "app: dotnet-guestbook-frontend"
+# - Pod exposes port 8080
+# - specify PORT environment variable to the container process
+# Syntax reference https://kubernetes.io/docs/concepts/configuration/overview/
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: dotnet-guestbook-backend
+  labels:
+    app: dotnet-guestbook
+    tier: backend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: dotnet-guestbook
+        tier: backend
+    spec:
+      containers:
+      - name: backend
+        image: dotnet-guestbook-backend
+        ports:
+        - name: http-server
+          containerPort: 8080

```

Comment:
I would expect the port number to be a member of a [configmap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) shared between the front end and backend pods. And probably for the debug port too.

Reply:
Yes, agree. We considered it along with kubernetes secrets for mongoDB and dropped for simplicity. All the samples (4 other languages) share the same yaml configurations with only object names being different (prefix for language)